src/components/SpacerView.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Animated, {useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+import PropTypes from 'prop-types';
+import * as StyleUtils from '../styles/StyleUtils';
+
+const propTypes = {
+    /**
+     * Should we show the spacer
+     */
+    shouldShow: PropTypes.bool.isRequired,
+
+    /**
+     * Array of style objects
+     * @default []
+     */
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+};
+
+const defaultProps = {
+    style: [],
+};
+
+function SpacerView({shouldShow, style}) {
```

Comment:
Please opt for [declaring default values when descructuring](https://github.com/namhihi237/App/blob/9ea79cdaf6b2ef4c49bdafc3cc03ad83fd74dcc7/contributingGuides/STYLE.md#L213-L238)